src/openrct2-ui/windows/StaffFirePrompt.cpp

```diff
@@ -18,10 +18,14 @@
 #include <openrct2/localisation/Formatter.h>
 #include <openrct2/localisation/Localisation.h>
 
+#pragma region WINDOW CONSTANTS
```

Comment:
Why this pragmas?

Reply:
I added this to be able to fold the code block. Removed in new commit.